README.md

```diff
@@ -269,3 +269,8 @@ Current OpenJDK implementation is based on using the CRIU project to create the
 [CRIU](https://github.com/CRaC/criu) hosts a few changes made to improve CRaC usability.
 
 You can read more about debugging C/R issues in your application in the [debug guide](./debugging.md).
+
+## Migration examples
+
+[Quarkus Super Heroes migration](super-heroes.md) shows a walkthrough for making an existing non-trivial application CRaC-able.
```

Comment:
Does it make sense to list it as a part of the "flow" https://github.com/CRaC/docs/blob/e21eb1c2fb0e4dcbdacc1e499422a959aee3c4b6/README.md#programmers-flow

Maybe another bullet along spring-boot?